### PR TITLE
Full Erc20 compatibility

### PR DESCRIPTION
**Pull request by nitika-goel**
In order to maintain 100% compatibility with ERC-20, the following changes have been made:
1. Removal of locked tokens from balances
2. Methods to get unlockable token balance
3. Methods to unlock tokens once the lock period has expired

### CODE REVIEW

File: test/LockableToken-creation_test.js [hunk 1]
```diff
@@ -12,6 +13,35 @@ contract('LockableToken', ([owner, receiver, spender]) => {
   const lockTimestamp = Number(new Date()) / 1000
   const approveAmount = 10
   const nullAddress = 0x0000000000000000000000000000000000000000
+  const increaseTime = function(duration) {
```
Comment (davesag) [inline test/LockableToken-creation_test.js:16]: do you need both this and the `timeTravel` function?

**Comment by maxsam4** [comment]
This can be closed in favor of PR #6

File: test/LockableToken-creation_test.js (hunk 1 above)
Comment (nitika-goel) [reply test/LockableToken-creation_test.js:16]: THanks @dave. Haven't used timeTravel. Will remove the include.

**Comment by nitika-goel** [comment]
Thanks @maxsam4 . Closing this PR.

**Pull request closed without merging**